src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+
+        return user == null ? false : user.getPassword().equals(password);
```

Comment:
let's make your code more readable:
```suggestion
        return user != null && user.getPassword().equals(password);
```